package.json

```diff
@@ -8,8 +8,8 @@
 	],
 	"scripts": {
 		"test": "npm run compile && tsc --project test/programs/typescript && jest",
-		"lint": "eslint --ext .js,.ts src test && prettier --check '**/*.{ts,js,json}'",
-		"format": "eslint --fix --ext .js,.ts src test && prettier --write '**/*.{ts,js,json}'",
+		"lint": "eslint --ext .js,.ts . && prettier --check '**/*.{ts,js,json}'",
+		"format": "eslint --fix --ext .js,.ts . && prettier --write '**/*.{ts,js,json}'",
```

Comment:
I have seen examples where the tsc generated output will currently fail to pass the lint / format checks, which is why I only had src and test folders specified.

My recollection is that it was complaining about generated variables not using camel-case.

Perhaps there is a tsc option to change it's output.

Reply:
This was fixed in #120.